metatron-app/metatron-discovery
# F#2132 detail explore metadata

### Description

Detail Explore Metadata 
Get TopUser, Histories, Recently Queries using api

**Related Issue** : 

#2132 

### How Has This Been Tested?

1. Using LNB go to explore data page
2. Click Metadata Card 
3. Check data is shown properly

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
